app/lib/frontend/backend.dart

```diff
@@ -329,6 +330,14 @@ class GCloudPackageRepository extends PackageRepository {
         rethrow;
       }
     });
+
+    storeFuture.then((PackageVersion pv) {
+      /// Notify analyzer services about a new version, and *DO NOT* do the
+      /// same with search service. The later will get notified after analyzer
+      /// ran the first analysis on the new version.
+      notifyAnalyzer(pv.packageName, pv.versionString);
+    }, onError: (_) => null);
+    return await storeFuture;
```

Comment:
No need to await the future, can just return it. But why doing this in the first place:

```dart
final PackageVersion pv = await db.withTransaction(....);
/// Notify analyzer services about a new version, and *DO NOT* do the
/// same with search service. The later will get notified after analyzer
/// ran the first analysis on the new version.
notifyAnalyzer(pv.package, pv.versionString);
return pv;